pkg/utils/cniconfig_test.go

```diff
@@ -20,17 +20,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"os"
 	"path/filepath"
-	"testing"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
-func TestCNIConfig(t *testing.T) {
```

Comment:
Is this change related?

Reply:
Both changes are required to pass CI because after the go.mod changes the following error is happen:

```
$ go test -v ./pkg/utils/
=== RUN   TestCNIConfig
Running Suite: CNI Config
=========================
Random Seed: 1645455922
Will run 10 of 10 specs

~~~~~~~~~~
Ran 10 of 10 Specs in 0.001 seconds
SUCCESS! -- 10 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestCNIConfig (0.00s)
=== RUN   TestNetworkAttachmentDefinition
Running Suite: Network Attachment Definition utils
==================================================
Random Seed: 1645455922
Will run 20 of 20 specs

~ Failure [0.000 seconds]
Netwok Attachment Definition manipulations
/home/nrt/tohayash/src/npwg/network-attachment-definition-client-s1061123/pkg/utils/net-attach-def_test.go:48
  test convertDNS [It]
  /home/nrt/tohayash/src/npwg/network-attachment-definition-client-s1061123/pkg/utils/net-attach-def_test.go:50

  You may only call BeforeEach from within a Describe, Context or When

  /home/nrt/tohayash/src/npwg/network-attachment-definition-client-s1061123/pkg/utils/cniconfig_test.go:37
------------------------------
~~~~~~~~~~~~~~~~~~~

Summarizing 1 Failure:

[Fail] Netwok Attachment Definition manipulations [It] test convertDNS
/home/nrt/tohayash/src/npwg/network-attachment-definition-client-s1061123/pkg/utils/cniconfig_test.go:37

Ran 20 of 20 Specs in 0.001 seconds
FAIL! -- 19 Passed | 1 Failed | 0 Pending | 0 Skipped
--- FAIL: TestNetworkAttachmentDefinition (0.00s)
FAIL
FAIL    github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/utils  0.018s
FAIL
```